stacker_blueprints/aws_lambda.py

```diff
@@ -280,11 +280,14 @@ def create_policy(self):
     def create_role(self):
         t = self.template
 
-        vpc_policy = NoValue
+        role_kwargs = {
```

Comment:
Another way to do this would be:

```python
role = iam.Role("Role", AssumeRolePolicyDocument= get_lambda_assumerole_policy())
if self.get_variables["VpcConfig"]
  iam.Policies = [ ... ]
```

End result is the same, so either way is fine.